aptos-move/framework/aptos-framework/sources/nonce_validation.move

```diff
@@ -0,0 +1,148 @@
+module aptos_framework::nonce_validation {
+    use aptos_std::table::{Self, Table};
+    use aptos_std::timestamp;
+    use aptos_std::big_ordered_map::{Self, BigOrderedMap};
+    use aptos_std::aptos_hash::sip_hash_from_value;
+    use aptos_std::option;
+    use aptos_std::vector;
+    friend aptos_framework::genesis;
+    friend aptos_framework::transaction_validation;
+
+    const NUM_BUCKETS: u64 = 50000;
+    const MAP_SWITCH_TIME: u64 = 75;
+    const MAX_EXPIRATION_TIME: u64 = 60;
```

Comment:
why u64 if u32 could handle?

Reply:
This is a constant not stored anywhere.
timestamp.move uses u64 when returning the current timestamp, so used the same datatype.